Sources/PixelEditor/PixelEditViewController.swift

```diff
@@ -126,26 +133,83 @@ public final class PixelEditViewController : UIViewController {
     action: #selector(didTapCancelButton)
   )
 
-  private let imageSource: ImageSource
+  /// Setting this to true will cause the UI to display a spinner and blurr the current image, while disabling the UI.
+  /// Set to false to remove
+  /// This is usefull in case you wish to present the PixelEditior while the image is still loading
+  public var isLoading: Bool {
+    get {
+      return loadingView != nil
+    }
+    set {
+      if newValue, self.isLoading == false {
```

Comment:
I think it would be better to create UI component with separating code to get better management.
Would you mind if I improve this code when I get time?